helm/values.yaml

```diff
@@ -148,14 +148,6 @@ configMap:
             endpoint: "0.0.0.0:14250"
           thrift_http:
             endpoint: "0.0.0.0:14268"
-      # Collect own metrics
```

Comment:
We do not want to use prometheus receiver on platform collector as we directly scrape metrics from this collector by prometheus, and if we collect this collector's own metrics, it will duplicate internal metrics when exposed by prometheus exporter.